bigchaindb/commands/bigchaindb.py

```diff
@@ -53,22 +48,16 @@ def run_show_config(args):
     print(json.dumps(config, indent=4, sort_keys=True))
 
 
-def run_configure(args, skip_if_exists=False):
-    """Run a script to configure the current node.
-
-    Args:
-        skip_if_exists (bool): skip the function if a config file already exists
-    """
+@configure_bigchaindb
```

Comment:
@muawiakh Could you please explain me the reasoning behind this?

Reply:
`configure_bigchaindb` updates `bigchaindb.config` with the variables from the environment. A similar behavior was triggered by [L80](https://github.com/bigchaindb/bigchaindb/pull/2121/files/96729c04a0ca24f09cedb3e12fa59c46e83e1d89#diff-67b248049e9050606ab8a4455c4f6397L80) with a difference: in this context `bigchaindb.config_utils.update` updated only the **local** `conf` variable, not the global one, and this made testing messy. The correct solution in this case is to call `configure_bigchaindb` that would update the global `bigchaindb.config`.